src/core/Notification/Notification.md

```diff
@@ -0,0 +1,103 @@
+```js
+import {
+  Notification,
+  Button,
+  Paragraph
+} from 'suomifi-ui-components';
+
+<>
+  <Notification closeText="Close" smallScreen>
+    The service will be temporarily unavailable on 5.6.2021 at 21.00 –
+    23.59 due to maintenance.
+  </Notification>
+  <Notification
+    status="error"
+    labelText="Lorem ipsum dolor sit"
+    smallScreen
+  >
+    Something went wrong. Please try again in a moment.
+  </Notification>
+  <Notification
+    closeText="Close"
+    smallScreen
+    labelText="Lorem ipsum dolor sit"
+    headingVariant="h3"
+    actionElements={<Button variant="secondary">Button</Button>}
+  >
+    The service will be temporarily unavailable on 5.6.2021 at 21.00 –
+    23.59 due to maintenance.
+  </Notification>
+  <Notification
+    closeText="Close"
+    smallScreen
+    labelText="Lorem ipsum dolor sit"
+    headingVariant="h2"
+    actionElements={
+      <>
+        <Button>Button</Button>
+        <Button variant="secondary">Button</Button>
+      </>
+    }
+  >
+    The service will be temporarily unavailable on 5.6.2021 at 21.00 –
+    23.59 due to maintenance.
+  </Notification>
+  <Notification closeText="Close">
+    <Paragraph>
```

Comment:
I'd drop the other paragraph from here to save space. One is enough to showcase using elements as the main content. We also want to showcase using the correct styling even when using elements as the main content, so I'd add `style={{fontSize: '16px'}}` for the paragraph.

Reply:
Fixed. Added fontSize.